pyvsystems/wrapper.py

```diff
@@ -24,9 +24,17 @@ def request(self, api, post_data=''):
                     return requests.post(url, data=post_data, headers=headers).json()
             else:
                 if self.timeout:
-                    return requests.get(url, headers=headers, timeout=self.timeout).json()
+                    req = requests.get(url, headers=headers, timeout=self.timeout).json()
+                    if (req.status_code != 200):
+                        raise NetworkException("Failed to get response in offline mode.")
```

Comment:
The offline mode feature in Python SDK does not means network issue. It uses for Cold Machine offline signature. Please change the exception message to "Failed to get response".